CONTRIBUTING.md

```diff
@@ -107,3 +107,38 @@ Icons are located in [packages/react/src/components/Icon/icons](./packages/react
 - Remove any namespace attributes (such as `xmlns:*`) from the SVG.
 - Save your icon in the above location with the proposed name of your icon, e.g. `[your-name].svg`. The `your-name` portion will be how the icon is used in the React Component, e.g. `<Icon type="your-name" />`.
 - Submit a new `feat` PR with your changes.
+
+## Breaking Changes
+
+Cauldron is constantly evolving, and it's inevitable that breaking changes will need to be introduced. Before a breaking change is released, other alternatives should be considered and allow for a deprecation period before the breaking change is released.
+
+### Components
+
+For cauldron's component library, any of the following would be considered breaking changes:
+
+* Renaming (exported name) or removing a Component
+* Changing a prop that would break usage of an existing component
+* Adding a new required prop to an existing component
+
+### Styles
+
+For cauldron's styles library, any of the following would be considered breaking changes:
+
+* Changing or removing a class name
+* Changing or removing a global css custom property name (see `variables.css`)
+* Changing or removing a component css custom property name
+
+In addition, changing a css custom property may be a breaking change for the following:
+
+* Changing a global css custom property value (see `variables.css`)
+* Changing a component css custom property value
+
+For the latter, the change must be reviewed by a member of the cauldron team to ensure there are no breaking changes.
+
+### Deprecating
+
+Before a breaking change can be released, the breaking change should be documented with the component, property, or style being changed getting flagged as _Deprecated_. This could mean warning the consumer that a component or property is now deprecated, or including a deprecated comment next to a css class name. This deprecation must exist for at least two months. If a change is additive (e.g. making a new property required), the new property must be optional until the deprecation period has passed.
```

Comment:
> If a change is additive (e.g. making a new property required), the new property must be optional until the deprecation period has passed.

nit: It seems odd to find this under a heading for "Deprecating" and calling it a "deprecation period", when a new required property isn't a deprecation

Reply:
Technically it is, because if you want to add a required property you're deprecating usage of the component without the required prop.

I think we should be able to avoid this scenario for the most part, I'm trying to envision the scenarios where we would add a new required prop and struggled to come up with good examples unless we completely missed something during the initial development of a component.

Reply:
> I think we should be able to avoid this scenario for the most part, I'm trying to envision the scenarios where we would add a new required prop and struggled to come up with good examples unless we completely missed something during the initial development of a component.

Yeah, I think this scenario is unusual.